packages/frontend/src/main/components/viewer/CommitInfoResource.vue

```diff
@@ -126,46 +125,36 @@ export default {
       return this.resource.data.commit
     },
     isolated() {
-      return (
-        this.viewerState.isolateValues.indexOf(
-          this.resource.data.commit.referencedObject
-        ) !== -1
+      if (!this.viewerState.currentFilterState?.isolatedObjects) return false
+
+      return this.viewerState.currentFilterState?.isolatedObjects?.includes(
+        this.resource.data.commit.referencedObject
       )
     },
     visible() {
-      return (
-        this.viewerState.hideValues.indexOf(
-          this.resource.data.commit.referencedObject
-        ) === -1
+      if (!this.viewerState.currentFilterState?.hiddenObjects) return true
+
+      return !this.viewerState.currentFilterState?.hiddenObjects?.includes(
+        this.resource.data.commit.referencedObject
       )
     }
   },
   methods: {
     isolate() {
       const id = this.resource.data.commit.referencedObject
-      if (this.isolated)
-        unisolateObjects({
-          filterKey: '__parents',
-          filterValues: [id]
-        })
-      else
-        isolateObjects({
-          filterKey: '__parents',
-          filterValues: [id]
-        })
+      if (this.isolated) {
+        unIsolateObjects([id], 'ui-iso', true)
```

Comment:
ui-iso and ui-vis repeat multiple times. do they have some sort of special meaning? if so, shouldnt they be constants?

Reply:
No special meaning really - it's a way of splitting up commands coming from different controls (model explorer, filters, selection) so the viewer filtering api can know whether to reset its internal state or not (e.g., model explorer show/hide filtering commands stack, but if a new show/hide command comes from, say, the selection info component - the prev visibility state should be cleared).